ocf_datapipes/utils/consts.py

```diff
@@ -132,17 +133,32 @@ def __getitem__(self, key):
 UKV_STD = _to_data_array(UKV_STD)
 UKV_MEAN = _to_data_array(UKV_MEAN)
 
-# These were calculated from 200 random init times (step 0s) from the MO global data
+# --- MO Global
+
 MO_GLOBAL_INDIA_MEAN = {
```

Comment:
I'm worried about breaking older models by doing this. 
I might splitl this into MO_GLOBAL_INDIA_MEAN and MO_GLOBAL_INDIA_NEW, The default being the orginal one, and the new one will need to be selected. This does make you think abit how we version control things a bit later in ocf-data-sampler